servers/graphql-kotlin-spring-server/src/test/kotlin/com/expediagroup/graphql/server/spring/FederationConfigurationTest.kt

```diff
@@ -79,6 +87,38 @@ class FederationConfigurationTest {
             }
     }
 
+    @Test
+    fun `verify federated schema execution with federated tracing`() {
+        contextRunner.withUserConfiguration(FederatedConfiguration::class.java)
+            .withPropertyValues(
+                "graphql.packages=com.expediagroup.graphql.server.spring",
+                "graphql.federation.enabled=true",
+                "graphql.federation.tracing.enabled=true"
+            )
+            .run { ctx ->
+                assertThat(ctx).hasSingleBean(GraphQL::class.java)
+                assertThat(ctx).hasSingleBean(FederatedTracingInstrumentation::class.java)
+
+                val graphql = ctx.getBean(GraphQL::class.java)
+                val federatedContext = MockFederatedGraphQLContext()
+
+                // TODO: This is failing until federation-jvm updates to graphql-java 16
```

Comment:
This is going to fail until the federation-jvm libraries is updated: https://github.com/apollographql/federation-jvm/pull/84